new_listings_scraper.py

```diff
@@ -53,19 +55,65 @@ def get_announcement():
     return latest_announcement['data']['catalogs'][0]['articles'][0]['title']
 
 
+def get_kucoin_announcement():
```

Comment:
Most of that seems identical to the way we check Binance. Would it make sense to extract some kind of `create_url(exchange)` function?

Reply:
Can definitely be consolidated I agree. I'll merge this in for now if all good, and will try and think of a way to reduce the amount of duplicated code.

Reply:
Fair enough. :+1: